### PR TITLE
Modules

### PR DESCRIPTION
**Pull request by Maedea9**
As requested in this project I have:
- Created a new directory and initialized it with git.
- Used npm init -y to create a package.json file
- Created index.js file and index.css
- Linked previous files to index.html
- Created a directory named modules.

I have recreated the app using:
- ES6 syntax
- Divided the code into modules and imported them into the index.js file
- All functionalities are working
- Used arrow functions
- Imported the current date below the navigation bar using Luxon.

### CODE REVIEW

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @Maedea9  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 

You have submitted a project, that doesn't have all sections of the README completed. Kindly make sure all related sections of this project are complete. For example, you're missing the `future features` part of the sections

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import Books from './modules/booksclass.js';
```
Comment (jasonfelice) [inline index.js:1]: - [x] The source of this file is missing. Please fix this issue before requesting the next review

File: index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
```
Comment (jasonfelice) [inline index.js]: - [ ] I'm getting a `404 Not Found` error for this line. It is possible that it is working on your system but not on others. Make sure to include luxon in your `package.json` so it installs whenever user runs `npm install`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  </div>
+  </main>
+  <!--Add a book form-->
+  <div class="form-container hide">
+    <h2 class="subtitle">Add a Book</h2>
+      <form class= "add-form" id="add-form">
+        <input type="text" required placeholder="Title of the book" id="title">
+        <input type="text" required placeholder="Author of the book" id="author">
+        <button type="submit" id="submit">Add</button>
+      </form>
```
Comment (jasonfelice) [inline index.html:48]: - [ ] [Optional] You should consider adding some kind of alert to notify the user about the action.

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @Maedea9, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
The Design is good✔️
Clean Code✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Maedea9) [reply index.js:1]: Hello @jasonfelice . A friend checked the code in his computer and says he had the same problem, that the module folder was empty. I believe I have fixed it now. Thanks

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @Maedea9 

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Descriptive PR ✔️
- Nice import/export flow✔️
- Good code structure ✔️

## Confirmed `luxon` in packages:
image


## Confirmed date View:
image




## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**